cmd/git/main.go

```diff
@@ -0,0 +1,356 @@
+// Copyright The Shipwright Contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package main
```

Comment:
Well, the package needs to be a `main` package since we start a program. Therefore, by popular convention I named the starter Go source file `main.go`, too.

Reply:
I agree. I think the missing piece was that this is intended to be an executable pkg, rather than an utility one.